routes/index.js

```diff
@@ -110,4 +108,94 @@ router.get(`/${APP_VERSION}/datastore_search`, async function (req, res, next) {
   }
 })
 
+//TODO finish this test function to manually check downloads
```

Comment:
still todo?

Reply:
Yes, the manual download is not working out of the box, I'd rather leave it for another PR.